src/conversion_helper.js

```diff
@@ -501,6 +507,17 @@ function roundToDecimalPlaces(number, places) {
   return (Math.round(number * multiplier)/multiplier).toFixed(places);
 }
 
+function ContainsArray(wrapper, main) {
```

Comment:
We have been using lower case letters for functions, perhaps rename this to `containsArray`?

Reply:
Yes sure will do. I actually mean't for you to make a review once I had done some changes, and just take a look right now to make sure I'm not breaking something major, but thanks anyway, I'm fairly confident that the main handling of ranges should be working now. So take a look now. 😄